test/cql-pytest/util.py

```diff
@@ -191,6 +191,12 @@ def cql_session(host, port, is_ssl, username, password):
         # so let's increase them to 60 seconds. See issue #11289.
         connect_timeout = 60,
         control_connection_timeout = 60,
+        # The default reconnection policy has a large maximum interval
+        # between retries (600 seconds). In tests that restart/replace nodes,
+        # where a node can be unavailable for an extended period of time,
+        # this can cause the reconnection retry interval to get very large,
+        # longer than a test timeout.
+        reconnection_policy = ExponentialReconnectionPolicy(1.0, 4.0)
```

Comment:
As I noted above this change shouldn't really help cql-pytest. And the comment suggests it does something which doesn't happen in cql-pytest.
I'm not strongly against this line (here, not in the other files where it's completely out of place), but I think you should consider if it's really useful, and if not, drop it.